.gitignore

```diff
@@ -91,7 +91,6 @@ c.out
 /cmd/weaviate-server/build_artifacts/
 
 esbackups/
```

Comment:
This is the only real change.

Reply:
@antas-marcin any rationale/historical context why we ignored this in the first place (given `testdata` is common way to provide data to go test)

Reply:
I think that's something from the old times, before `v1` version of Weaviate, Weaviate was built on top of Elasticsearch and since this folder has `es*` prefix then I think it was used long time ago